torchao/prototype/smoothquant/api.py

```diff
@@ -120,6 +137,7 @@ def _smooth_quant_transform(
     qw = to_weight_tensor_with_linear_activation_scale_metadata(
```

Comment:
we should not be using this, please check awq on how this should be implemented in the new stack: https://github.com/pytorch/ao/blob/08e5e203f1074a1663fe8293e184b252ea359f99/torchao/prototype/awq/api.py#L108-L113